app/com/linkedin/drelephant/analysis/HDFSContext.java

```diff
@@ -41,11 +43,24 @@ private HDFSContext() {
    */
   public static void load() {
     try {
-      HDFS_BLOCK_SIZE = FileSystem.get(new Configuration()).getDefaultBlockSize(new Path("/"));
+      HDFS_BLOCK_SIZE = minOfChildSystemsBlockSize(FileSystem.get(new Configuration()));
```

Comment:
I would add a comment that 
1) this is the default value and applications may override it
2) the value may be different between services but this estimation is good enough for now